### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by bbergshaven**
Publish V1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
 ![Clarify Insert Node](https://github.com/searis/node-red-contrib-clarify/blob/master/examples/random-data-example.png?raw=true)
 
 Any questions? Send us an email on support@clarify.us
+
+## Changelog
+
+From 1.0.0 we will not do breaking changes to the overall msg.format. 
+
+### 1.0.0 - Breaking changes from -beta.X
```
Comment (smyrman) [inline README.md:32]: Nice change log.

But should we do another beta?

I don't think we want to do any more changes to the input structure, but I don't think we have yet nailed the output yet.

Comment (smyrman) [reply README.md:32]: #37

File: README.md [hunk 2]
```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
 ![Clarify Insert Node](https://github.com/searis/node-red-contrib-clarify/blob/master/examples/random-data-example.png?raw=true)
 
 Any questions? Send us an email on support@clarify.us
+
+## Changelog
+
+From 1.0.0 we will not do breaking changes to the overall msg.format.
```
Comment (smyrman) [inline README.md:30]:
```suggestion
From v1.0.0-beta4 we do not expect to do any breaking changes to the input message format. However we might add new optional fields. As for the block output, development is still ongoing, and breaking changes must be expected before the final v1.0.0 release.
```

Comment (smyrman) [inline README.md:30]:
```suggestion
## Backwards compatibility

From v1.0.0-beta.4 we do not expect to do any breaking changes to the insert node input message format. However we might add new optional fields.

As for the insert node _output_ message format, development is still ongoing, and breaking changes must be expected before the final v1.0.0 release. The number of outputs may also change.

## Changelog

The changelog is introduced from `v.1.0.0-beta.4`, and describe changes from  `v.1.0.0-beta.3`.

```

File: README.md [hunk 3]
```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
...
+
+From 1.0.0 we will not do breaking changes to the overall msg.format. 
+
+### 1.0.0 - Breaking changes from -beta.X
+
+Updated the format of the messages according to this proposal: https://github.com/searis/node-red-contrib-clarify/issues/28
+
+- The Input ID is put in `msg.topic`
+- The signal meta data is moved out of the payload to `msg.signal`
+- `msg.payload.series` is renamed to `msg.payload.values`
```
Comment (smyrman) [inline README.md:38]: Wasn't it inside a `data` sub-structure before?
```suggestion
- `msg.payload.data.times` is renamed/moved to `msg.payload.times`
- `msg.payload.data.series` is renamed/moved to `msg.payload.values`
```

**Review by khtufteland — APPROVED** [review]

**Review by smyrman — CHANGES_REQUESTED** [review]
See comments.

File: README.md [hunk 4]
```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
...
+
+New message format:
+```json
```
Comment (smyrman) [inline README.md:41]: Try `js`

````suggestion
```js
````

File: README.md [hunk 5]
```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
...
+msg:
+  topic: "<Input ID>"
+  payload:
+    times: ["<timestamp>", ...]
+    values: [(<number>||null), ...]
+  signal: <Signal> # Match https://docs.clarify.us/reference#signal
```
Comment (smyrman) [inline README.md:47]:
```suggestion
  signal: <Signal> // Match https://docs.clarify.us/reference#signal
```

**Review by smyrman — APPROVED** [review]

**Pull request merged**